style.css

```diff
@@ -124,6 +124,96 @@ header {
   opacity: 0.5;
 }
 
+@media only screen and (min-width: 768px) {
+  header {
+    position: absolute;
+    width: 1440px;
+    top: 1px;
+    background: #fff;
+    margin-left: 250px;
+  }
+
+  #logo {
+    width: 76px;
+    height: 20px;
+    margin-left: 150%;
+  }
+
+  .header {
+    position: relative;
+    width: 1440px;
+    height: 718px;
+    margin-left: 250px;
+    background-color: #fff;
+    background-image: url(images/bg/Header\ new\ bg.png);
+    background-repeat: no-repeat;
+    background-size: 1260px;
+    background-position-x: 70px;
+    border-bottom-left-radius: 60px;
+  }
+
+  .nav_links li a {
+    font-size: 15px;
+  }
+
+  .nav_links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 25px;
+    margin-right: 10%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  /* .Header container */
+  .header_container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    position: absolute;
+    width: 920px;
+    height: 268px;
+    left: 260px;
+    top: calc(50% - 268px / 2);
+  }
+
+  .header_container h1 {
+    width: 420px;
+    height: 122px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .header_container p {
+    width: 920px;
+    height: 98px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+}
```

Comment:
- [ ] Great work on designing the desktop version of your portfolio :+1: :100:. However, as per the project requirements, the mobile version needs to be responsive up to the desktop screen size, and the desktop version needs to be fully responsive without any design errors in between. Unfortunately, there are some elements in the video that have a fixed width and are not responsive, which need to be addressed. Therefore, please ensure that all elements in your portfolio are responsive and without any design errors in different screen sizes :+1:.

https://user-images.githubusercontent.com/105117832/235990022-119e07a1-30e9-443f-b32e-41dc96b76b89.mp4

- [ ] In addition, I have noticed that some elements of your project are not properly centered and are slightly aligned towards the right side. This can make your website look unprofessional and less visually appealing. It is important to make sure that all the elements are properly aligned and centered to create a better user experience :+1:.